templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -534,13 +534,13 @@ Resources:
               VolumeType: "gp2"
         IamInstanceProfile:
           Arn:
-            Fn::ImportValue: !Sub "${Environment}${ApplicationName}WebInstanceProfile"
+            Fn::ImportValue: !Sub "${Environment}${ApplicationName}{% if layer != 'Solr' %}Web{% endif %}InstanceProfile"
```

Comment:
Should it be if == 'Web', add Web ? what about batch?

Reply:
Batch servers have the same access as web. They have mostly the same needs; the same SSM Parameter Store values to configure the plugins, the same access to the database and S3 filestore, etc.

Solr is special because it doesn't need as much SSM access and does need to update internal Route 53.